exonum/src/encoding/tests.rs

```diff
@@ -144,6 +148,48 @@ fn test_i64_segment() {
     assert_write_check_read(dat, 8);
 }
 
+#[cfg(feature="float_serialize")]
```

Comment:
Maybe this should be moved into the `float` module too?..

@vldm What do you think?